core/src/components/header/header.tsx

```diff
@@ -57,14 +57,13 @@ export class Header implements ComponentInterface {
     // Determine if the header can collapse
     const hasCollapse = this.collapse === 'condense';
     const canCollapse = (hasCollapse && getIonMode(this) === 'ios') ? hasCollapse : false;
-
     if (!canCollapse && this.collapsibleHeaderInitialized) {
       this.destroyCollapsibleHeader();
     } else if (canCollapse && !this.collapsibleHeaderInitialized) {
       const tabs = this.el.closest('ion-tabs');
       const page = this.el.closest('ion-app,ion-page,.ion-page,page-inner');
 
-      const pageEl = (tabs) ? tabs : (page) ? page : null;
+      const pageEl = (tabs) ? this.el.closest('ion-tab') : (page) ? page : null;
```

Comment:
`ion-tab` is not a component thats available, so `this.el.closest('ion-tab')` will always return `null` here.

Reply:
Whoops, `ion-tab` is a component in vanilla JS but not in `@ionic/angular`. Will fix, thanks.